frappe/desk/doctype/auto_repeat/auto_repeat.py

```diff
@@ -38,6 +38,7 @@ def on_submit(self):
 	def on_update_after_submit(self):
 		self.validate_dates()
 		self.set_next_schedule_date()
+		self.get_auto_repeat_schedule()
```

Comment:
sorry did not see this earlier. This should not be saved in the document, but just queried at the time of view.

Call this from `onload` method. 

```
def onload(self):
  self.set_onload("auto_repeat_schedule", self.get_auto_repeat_schedule())
```